cloud/services/virtualmachines/virtualmachines.go

```diff
@@ -60,7 +63,16 @@ func (s *Service) Get(ctx context.Context, spec interface{}) (interface{}, error
 		return vm, err
 	}
 
-	return converters.SDKToVM(vm)
+	convertedVM, err := converters.SDKToVM(vm)
```

Comment:
why not inside SDKToVM?

Reply:
Chose to do it inside of virtualmachines.go because the SDKToVM function doesn't have access to the scopes necessary to lookup the network interfaces and the public ip addresses.